sdrbase/util/aprs.cpp

```diff
@@ -988,3 +998,128 @@ bool APRSPacket::parseTelemetry(QString& info, int& idx)
     else
         return false;
 }
+
+bool APRSPacket::parseMicE(QString& info, int& idx, QString& dest)
+{
+    // Mic-E Location data is encoded in the AX.25 Destination Address
+    if (dest.length() < 6)
+        return false;
+
+    // Mic-E Information data is 8 bytes minimum, 13-14 with altitude
+    if (info.length() < idx+8)
+        return false;
+
+    QString latDigits = '';
+    QString messageBits = '';
```

Comment:
I wasn't sure if this should be a new `m_mice_messagebits` variable or if I should just shove it in comments somewhere. It's a lookup table with 7 options plus 1 emergency tag.

Reply:
single quotes are for character literals. "" is for strings.

Reply:
Fixed